rbac/management/cross_account_request/model.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Model for cross account access request management."""
+import logging
+from uuid import uuid4
+
+from django.db import models
+from django.utils import timezone
+from management.rbac_fields import AutoDateTimeField
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+STATUS_LIST = ["requested", "cancelled", "approved", "denied"]
+
+
+class CrossAccountRequest(models.Model):
+    """Cross account request."""
+
+    request_id = models.UUIDField(default=uuid4, editable=False, unique=True, null=False, primary_key=True)
+    target_account = models.CharField(max_length=15)
+    requestor = models.CharField(max_length=150)
```

Comment:
Is `requestor` meant to be the TAM, as a request being initiated by the employee? Will we need to tie this to a principal?

Reply:
Yes. Updated using requestor_account and requestor principal